persistence/sql/persister_session.go

```diff
@@ -58,6 +58,7 @@ func (p *Persister) GetSession(ctx context.Context, sid uuid.UUID, expandables s
 		s.Identity = i
 	}
 
+	s.Active = s.IsActive()
```

Comment:
Do we need this because IsActive returns false when expired, but Active might not be false?

Reply:
I think yes